xml/depl_maintenance.xml

```diff
@@ -508,6 +508,262 @@
+      </para>
+     </step>
+     <step>
+      <para>
+        To prevent the &postgres;-related chef code from running, unassign the values
+	from <literal>database-server</literal> role in the <guimenu>Database</guimenu>
+       &barcl;, and apply the &barcl;.
+      </para>
+     </step>
+     <step>
+      <para>
+       Start chef-client services on the nodes again.
+      </para>
+     </step>
+     <step>
+      <para>
+       Stop &postgres; on the &dbnode;s. Uninstall &postgres;packages.
+      </para>
+      <substeps>
+       <step>
+	<para>
+	 To stop the <literal>postgresql</literal> service, run the following
+	 command on each cluster node:
+	</para>
+<screen>crm resource stop postgresql
+crm resource stop fs-postgresql
+crm resource stop drbd-postgresql</screen>
+        <para>
+	 Run the last command only if the previous setup used DRBD.
+	</para>
+       </step>
+       <step>
+	<para>
+	 Remove the packages on all cluster nodes:
+	</para>
+	<screen>zypper rm postgresql94 postgresql94-server</screen>
+       </step>
+       <step>
+	<para>
+	 If you choose not to upgrade to &productname; 8 right away, delete unused pacemaker resource from each cluster node:
+	</para>
+<screen>crm conf delete drbd-postgresql
+crm conf delete fs-postgresql
+crm conf delete postgresql</screen>
+       </step>
+      </substeps>
+     </step>
+     <step>
+      <para>
+       While you can continue using DRBD with &rabbit;, it is recommended to
+       switch to the clustering setup. If you don't make the switch at this
+       point, the upgrade to &productname; 8 will do this automatically. If you
+       choose to switch to the clustering setup, stop DRBD and uninstall
+       DRBD-related packages using the following command:
+      </para>
+<screen>&prompt.sudo;zypper rm drbd drbd-utils</screen>
+     </step>
+     <step>
+      <para>
+       Reclaim the disk space sued by &crow; for DRBD. To do this, edit the node data using <systemitem>knife</systemitem>:
+      </para>
+<screen>&prompt.user;knife node edit -a <replaceable>DRBD_NODE</replaceable></screen>
+      <para>
+        Search for <literal>claimed_disks</literal> and remove the entry with owner set to <literal>LVM_DRBD</literal>.
+      </para>
+     </step>
+    </procedure>
+   </sect3>
+   <sect3 xml:id="sec.postgre-mariadb.upgrade.scenario2">
+    <title>HA Setup with Two &contrnode;s without DRBD</title>
+    <para>
+     Extend the cluster with additional node that without any role assigned to
+     it. In &crow; &wi;, switch to the <guimenu>&pacemaker;</guimenu> &barcl;,
+     add the <literal>pacemaker-cluster-member</literal> role to the new node,
+     and apply &barcl;. Proceed with the migration procedure as described in
+     <xref linkend="sec.postgre-mariadb.upgrade.scenario1"/>.
+    </para>
+   </sect3>
+   <sect3 xml:id="sec.postgre-mariadb.upgrade.scenario3">
+    <title>HA Setup with 2 &contrnode;s and DRBD Used for Database or &rabbit;</title>
```

Comment:
Please merge with the previous section and only mention that point 2 ( allow_larger_cluster part) is specific to DRBD setup

Reply:
Fixed.